### ISSUE TITLE
Advanced Example: Cannot import multiples coordinates

### ISSUE THREAD

**Issue by ChiXuUnited** [gmaps-marker-clusterer/gmaps-marker-clusterer#61]
Hello everyone,

Nice job with the well-documented repository! 

I am a rookie, and I have been struggling to import my data (around 200 coordinates (latitude, longitude)) to the advance example you have in the repository.

I copied and pasted the HTML code of the advanced example, but cannot figure out how to import all my coordinates to the code.

I tried putting everything in the following format: "var marker = new google.maps.Marker({position: new google.maps.LatLng(-6.369028, 34.888822)});"

but the map wouldn't get uploaded correnctly.

So basically I wish to have a map with the following effect 
![cluteringmarker](https://user-images.githubusercontent.com/35962736/35860897-16774efc-0b46-11e8-9bf0-c289e49ab2c4.png)
, but couldn't figure out how to import coordinates.

Thanks in advance!

**Comment by XmlmXmlmX** [comment]
Hi @ChiXuUnited,

thank you for your question.

The trick is, to add all locations (markers) to an `markers` array.

```
var markers = []; // Create a markers array to collect all markers

var marker = new google.maps.Marker({ // Create a new marker with it's lat/lng
  position: new google.maps.LatLng(51.5074, 0.1278)
});
markers.push(marker); // Push each marker to the markers array

var options = {
  imagePath: 'images/m'
};

var markerCluster = new MarkerClusterer(map, markers, options); // Create a new Marker Clusterer Object
```

**Comment by ChiXuUnited** [comment]
Hello Sebastian @XmlmXmlmX,

I am still getting nothing here: I added the code to the array as you described, 
here is the code I came up with: 

        var markers = [];
////// I kept the original code of advanced exemple.html in between
          var latLng = new google.maps.LatLng(data.photos[i].latitude,
              data.photos[i].longitude)
          var marker = new google.maps.Marker({
new google.maps.LatLng(12.238333, -1.561593);
new google.maps.LatLng(11.825138, 42.590275)
          });
          markers.push(marker);
        }
/////// original code here again, until:
        markerClusterer = new MarkerClusterer(map, markers, {
          maxZoom: zoom,
          gridSize: size,
          styles: styles[style],
          imagePath: '../images/m'
        });
      }
//////// then some more original code 

Then I saved the code in a html file, but the whenever I open the html file, I've got:
![2018-02-07_105003](https://user-images.githubusercontent.com/35962736/35909798-e797f866-0bf4-11e8-85ba-08d39c7fb131.png)

Should I send you my entier code so that you can have a closer look?

**Comment by ChiXuUnited** [comment]
Hello again Sebastian @XmlmXmlmX 
 
I managed to solve (partly) the previous issue, now I succeed in loading the map! (feeling very proud now)

Now the issue is that I still don't see any marker on the map.

Will continue debugging the code, but will certainly appreciate it greatly if you can give me any advise

**Comment by ChiXuUnited** [comment]
here is what it look like now
![2018-02-07_120717](https://user-images.githubusercontent.com/35962736/35913302-7795202e-0bff-11e8-8df4-144b3e1ae3b3.png)

**Comment by XmlmXmlmX** [comment]
Have a look here: https://jsfiddle.net/Mescaline/yazt8j34/3/

**Comment by ChiXuUnited** [comment]
Hello again Sebastien @XmlmXmlmX 

The problem I am encountering is to have the beautiful layout of your advanced example, while importing my database to the map.

I still couldn't succeed, because I cannot figure out how and where to put my coordinates in the code of "advanced example.html".

Is there any advise you can give me on this, Sebastien?
